README.md

```diff
@@ -46,6 +46,7 @@
 | [Transact-SQL (T-SQL)](https://github.com/Ebazhanov/in-quiz-questions/blob/master/t-sql/t-sql-quiz.md) |  with answers  |
 | [WordPress](https://github.com/Ebazhanov/in-quiz-questions/blob/master/wordpress/wordpress-quiz.md) |  answers required  |
 | [XML](https://github.com/Ebazhanov/in-quiz-questions/blob/master/xml/xml-quiz.md) |  answers required |
+| [Android] |  answers required |
```

Comment:
@surajsahani sorry I didn't mention but I already added a link to android(when I merged first time your first PR), please check it out. So could you please remove this line
```suggestion

```

Reply:
okay.